cli/mfg.c

```diff
@@ -404,12 +404,50 @@ static int image_select(int argc, char **argv)
 	return ret;
 }
 
+static int boot_resume(int argc, char **argv)
+{
+	const char *desc = "Resume device boot process (BL1 and BL2 only)";
+	int ret;
+	enum switchtec_boot_phase phase_id;
+
+	static struct {
+		struct switchtec_dev *dev;
+	} cfg = {};
+	const struct argconfig_options opts[] = {
+		DEVICE_OPTION_NO_PAX,
+		{NULL}
+	};
+
+	argconfig_parse(argc, argv, desc, opts, &cfg, sizeof(cfg));
+
+	ret = switchtec_get_device_info(cfg.dev, &phase_id, NULL, NULL);
+	if (ret < 0) {
+		switchtec_perror("mfg boot-resume");
+		return ret;
+	}
+	if (phase_id == SWITCHTEC_BOOT_PHASE_FW) {
+		fprintf(stderr,
+			"This command is only available in BL1 or BL2!\n");
+		return -1;
+	}
+
+	ret = switchtec_boot_resume(cfg.dev);
+	if (ret < 0) {
+		switchtec_perror("mfg boot-resume");
+		return ret;
+	}
+
+	return 0;
+}
+
 static const struct cmd commands[] = {
 	{"ping", ping, "Ping firmware and get current boot phase"},
 	{"info", info, "Display security settings"},
 	{"mailbox", mailbox, "Retrieve mailbox logs"},
 	{"image_list", image_list, "Display active image list (BL1 only)"},
 	{"image_select", image_select, "Select active image index (BL1 only)"},
+	{"boot_resume", boot_resume,
+		"Resume device boot process (BL1 and BL2 only)"},
```

Comment:
Also a nit I didn't notice before: these should all use the CMD macro given the command line name and function name are the same.

Reply:
OK. For now I plan to keep this as-is and change all of them in one PR later, when all commands are checked in. Makes it easy for review as well.